src/Framework/AspNetCoreAnalyzers/src/Analyzers/RouteHandlers/DetectMisplacedLambdaAttribute.cs

```diff
@@ -73,5 +68,38 @@ static bool IsInValidNamespace(INamespaceSymbol? @namespace)
 
             return false;
         }
+
+        static IMethodSymbol? GetReturnedInvocation(IBlockOperation blockOperation)
+        {
+            foreach (var op in blockOperation.ChildOperations.Reverse())
```

Comment:
Why do we search the child operations in reverse here?

Reply:
I figured it made sense to search from the end of the method for return operations instead of the start because I'd expect most returns to be nearer to the end. Multiple return operations are an edge case this analyzer never really accounted for anyway.